### PR TITLE
Add configuration for new local-links-manager app

### PR DESCRIPTION
**Pull request by brenetic**
- Add puppet configuration for new local-links-manager app to help manage the local links within GOV.UK so that they can be used for local transactions.
- Add postgresql configuration for the local_links_manager app.

These are all base settings and will no doubt change as the app increments.

Relies on these PR's:
https://github.gds/gds/deployment/pull/929
https://github.gds/gds/deployment/pull/937

[Trello card](https://trello.com/c/cAioCrSk/316-get-a-new-local-links-manager-app-up-and-running-3)

### CODE REVIEW

File: modules/govuk/manifests/apps/local_links_manager.pp [hunk 1]
```diff
@@ -0,0 +1,19 @@
+# == Class: govuk::apps::local_links_manager
+#
+# App to manage the local links within GOV.UK so that
+# they can be used for local transactions.
+# https://github.com/alphagov/local-links-manager
+#
+class govuk::apps::local_links_manager(
+  $port = 3120,
```
Comment (issyl0) [inline modules/govuk/manifests/apps/local_links_manager.pp]: As of https://github.gds/gds/development/pull/299, 3120 is taken by Sidekiq Monitoring for Imminence.

File: hieradata/class/backend.yaml [hunk 2]
```diff
@@ -38,6 +38,7 @@ govuk::node::s_base::apps:
   - hmrc_manuals_api
   - imminence
   - kibana
+  - local-links-manager
```
Comment (sihugh) [inline hieradata/class/backend.yaml]: This needs to be `local_links_manager` to match https://github.com/alphagov/govuk-puppet/blob/add-local-links-manager-app/modules/govuk/manifests/apps/local_links_manager.pp#L12

File: modules/govuk/manifests/apps/local_links_manager.pp (hunk 1 above)
Comment (brenetic) [inline modules/govuk/manifests/apps/local_links_manager.pp]: :+1:   Swapped it out: https://github.gds/gds/development/pull/301

File: hieradata/class/backend.yaml (hunk 2 above)
Comment (brenetic) [inline hieradata/class/backend.yaml]: :+1:  Will update

**Comment by boffbowsh** [comment]
Looks good, just waiting for a change in deployment repo :+1:

**Comment by boffbowsh** [comment]
Still :+1:, will merge on Tues

**Pull request merged**